doc/neps/nep-0026-missing-data-summary.rst

```diff
@@ -704,14 +706,14 @@ risk of reducing developer contribution.
 References and Footnotes
 ------------------------
 
-`NEP 12`_ describes Mark's NA-semantics/mask implementation/view based mask
+:ref:`NEP12` describes Mark's NA-semantics/mask implementation/view based mask
 handling API.
 
-`NEP 24`_ ("the alterNEP") was Nathaniel's initial attempt at separating MISSING
+:ref:`NEP24` ("the alterNEP") was Nathaniel's initial attempt at separating MISSING
 and IGNORED handling into bit-patterns versus masks, though there's a bunch
 he would change about the proposal at this point.
 
-`NEP 25`_ ("miniNEP 2") was a later attempt by Nathaniel to sketch out an
+:ref:`NEP25` ("miniNEP 2") was a later attempt by Nathaniel to sketch out an
```

Comment:
This does change the rendering by adding the full title to the text, but I think that's a good thing here:

<img width="821" alt="image" src="https://user-images.githubusercontent.com/98330/83349355-9f08c380-a334-11ea-9db8-1cc3e6edd0f7.png">